### ISSUE TITLE
Feature request: Glossary for individual posts

### ISSUE THREAD

**Issue by MiMoHo** [pojome/elementor#5951]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor (Elementor Pro)

**What problem is your feature request going to solve? Please describe.**
I wish to set up a glossary for an extensive post deeply diving into a topic requiring special wording.

**Describe the solution you'd like**
Want to link words to a glossary that I can add at the top or bottom of a page/article. The reader should be able to easily click on a highlighted word getting redirected to the glossary without scrolling and hoping to find what he's looking for.

**Describe alternatives you've considered**
Add text fields and separator lines to set up a glossary manually without links to the relevant words on the page.

**Comment by bainternet** [comment]
@MiMoHo 

Can you provide examples?

**Comment by MiMoHo** [comment]
There's an example for a [tooltip plugin](https://essential-addons.com/elementor/tooltip) out there already that should be easy to integrate into Elementor Pro!
I'd prefer a solution like the plugin [CM Tooltip Glossary](https://de.wordpress.org/plugins/enhanced-tooltipglossary) that lets me create tooltips and a universal glossary as explained on [this](https://www.cminds.com/build-glossary-wordpress) demo site. However, a glossary for _individual_ posts would be more useful when a website consists of blog posts dealing with different things compared to a website dedicated to just one single topic where a global glossary may be relevant.
In short: I want to highlight text in editing mode in Elementor and make an explanation pop up for certain words or a sentence when the reader hovers over it. Then collect all explained words in a glossary at the bottom of the post.
